src/Symfony/Component/Form/Extension/Core/Type/NumberType.php

```diff
@@ -35,9 +36,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $precision = function (Options $options) {
```

Comment:
this is a closure getting the scale, not the precision, so ``$scale`` would be better